static/swagger-ui-3.52.0-dist/index.html

```diff
@@ -0,0 +1,60 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
Is this file needed? It looks like just a sample

Reply:
this is the base page of swagger ui, it is customized to a django template openapi.html, it is preferred to keep all resources under this folder - they are all unpacked from the release zip of swagger ui dist (tarball or zip), and take only the contents under dist sub folder.